### PR TITLE
feat: display `requested amount` and `recipient` fields inside Community Pool Spend proposal details page

### PR DESCRIPTION
**Pull request by MonikaCat**
## Description

Closes: #XXXX
[BDU-675](https://forbole.atlassian.net/browse/BDU-675)

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] ran linting
- [x] wrote tests where necessary
- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [x] targeted the correct branch
- [x] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed
- [x] added an entry to the `CHANGELOG.md` file

### CODE REVIEW

**Comment by teamchong** [comment]
I want to clarify on how we add a new field `proposalType: string` to [general_types.ts](https://github.com/forbole/big-dipper-2.0-cosmos/blob/bdu-675-community-pool-spend-proposals/packages/ui/src/graphql/types/general_types.ts#L11584)
is this the steps to add a new field?
- on [proposal_details.graphql](https://github.com/forbole/big-dipper-2.0-cosmos/blob/bdu-675-community-pool-spend-proposals/packages/ui/src/graphql/general/proposal_details.graphql#L10) add the field `proposalType: proposal_type`
- if the other chain had the same change, use find and replace to update all of them
- run `yarn graphql:codegen` to update `general_types.ts`

**Comment by teamchong** [comment]
https://pr-afzilu1v2-bigdipper.vercel.app/cosmos

**Comment by teamchong** [comment]
Tested ACK

File: packages/ui/src/screens/proposal_details/utils.tsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ export const getProposalType = (proposalType: string) => {
     type = 'softwareUpgradeProposal';
   }
 
-  if (proposalType === '/cosmos.upgrade.v1beta1.CommunityPoolSpendProposal') {
+  if (proposalType === '/cosmos.distribution.v1beta1.CommunityPoolSpendProposal') {
```
Comment (teamchong) [inline packages/ui/src/screens/proposal_details/utils.tsx:15]: just to confirm that this is an intended change, not merge by my mistake, right?

Comment (MonikaCat) [reply packages/ui/src/screens/proposal_details/utils.tsx:15]: @teamchong hey yes it is, the old path was wrong

**Review by teamchong — APPROVED** [review]
thanks

**Pull request merged**